lib/active_merchant/billing/gateways/barclaycard_smartpay.rb

```diff
@@ -37,7 +37,7 @@ def authorize(money, creditcard, options = {})
         post[:card] = credit_card_hash(creditcard)
         post[:billingAddress] = billing_address_hash(options) if options[:billing_address]
         post[:deliveryAddress] = shipping_address_hash(options) if options[:shipping_address]
-        add_3ds(post, options) if options[:execute_threed]
+        add_3ds(post, options)
```

Comment:
We don't need the `:execute_threed`?

Reply:
good question - i was mostly leaning on what's happening for adyen, which, in the `add_3ds` method will add 3ds2 browser info if that gets passed in (whether it receives the `execute_threed` flag or not), or will check that a 3ds flag is present before falling back to adding 3ds1 data

Reply:
👍 Might be something to monitor when it goes in production.